web-animations-2/Overview.bs

```diff
@@ -2460,13 +2480,42 @@ given values of <var ignore=''>behavior</var>:
 
 Issue: Do we need a proper definition for "behavior types primary/inverse"?
 
-### Animation Trigger Active Interval ### {#trigger-active-interval}
+#### Event-Based Animation Trigger Behavior #### {#event-trigger-behaviors}
```

Comment:
Rather than mix the timeline and event based trigger sections it seems like we should have a different top-level section for timeline vs event-based triggers, e.g.

* Animation Triggers
  * Overview
  * Event-based
    * Behavior
  * Timeline-based
    * Behavior
    * Range
    * State
    * etc
  * Updating

Reply:
Done.